app/views/static_pages/index.html.erb

```diff
@@ -78,5 +64,19 @@
         <% end %>
       </table>
     </div>
+    <div id="twitter_feed">
+      <a class="twitter-timeline"
+         width="600"
+         height="250"
+         href="https://twitter.com/hashtag/hubstub"
+         data-widget-id="567540234242367488">
+         #hubstub Tweets</a>
+    <script>
+      !function(d,s,id){
+        var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){
+          js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);
+        }}(document,"script","twitter-wjs");
+    </script>
```

Comment:
I was actually thinking about doing this too. I think it looks better this way. :herb: